test/e2e/autotls/auto_tls_test.go

```diff
@@ -84,11 +66,10 @@ type config struct {
 	SetUpDNS                      string `split_words:"true" required:"false"`
 }
 
-type autoTLSClients struct {
-	clients  *test.Clients
-	cmClient cmclientset.Interface
-}
-
+// To run this test locally with cert-manager, you need to
+// 1. Install cert-manager from `third_party/` directory.
+// 2. Run the command below to do the configuration:
+// kubectl apply -f test/config/autotls/certmanager/selfsigned/
 func TestPerKsvcCert_localCA(t *testing.T) {
```

Comment:
This test name will be ignored most likely.
`TestPerKsvcCertLocalCA` is a better name.

Reply:
Done